app/services/network_topology_service.rb

```diff
@@ -94,7 +94,7 @@ def entity_status(entity)
     when Vm
       entity.power_state.capitalize
     when ManageIQ::Providers::NetworkManager
-      entity.authentications.empty? ? 'Unknown' : entity.authentications.first.status.capitalize
+      entity.authentications.blank? ? 'Unknown' : entity.authentications.first.status.try(:capitalize)
```

Comment:
unfortunately authentications.nil?, if the delegated object is not there.

Hm, for delegation to active record relation, it would be nice if we could somehow return empty arel, instead of nil.